lib/evss/configuration.rb

```diff
@@ -3,8 +3,6 @@
 
 module EVSS
   class Configuration < Common::Client::Configuration::REST
-    DEFAULT_TIMEOUT = 15
```

Comment:
Do we not still need a default value in the event that the respective `Settings.evss.<SERICE>.timeout` is not set?